html/mix-manifest.json

```diff
@@ -1,13 +1,10 @@
 {
-    "/js/app.js": "/js/app.js?id=eafcc78bac952df12bbc",
```

Comment:
this is wrong

Reply:
what about now?

Reply:
You're removing multiple lines that needs to be there

Reply:
i did only :
`su librenms -c "./lnms translation:generate"`
which (re)generates/refreshes mix-manifest.json
nothing more ...

Reply:
could be a bug with that process